### PR TITLE
BZ1851483: Preparing to reinstall a cluster on bare-metal

### PR DESCRIPTION
**Pull request by tmulquee**
Version(s):
From Version 4.9 to 4.11 and beyond. 

Issue:
https://bugzilla.redhat.com/show_bug.cgi?id=1851483

Link to docs preview: 

http://file.emea.redhat.com/tmulquee/BZ1851483/installing/installing_bare_metal_ipi/ipi-install-installation-workflow.html#ipi-preparing-reinstall-cluster-bare-metal_ipi-install-installation-workflow

Additional information:
This PR is the parent of 3 child PRs due to divergence of terminology in 4.6, 4.7, and 4.8 (Control Plane node / Control Plane (also known as master) node).

https://github.com/openshift/openshift-docs/pull/46813 (for Version 4.6)

https://github.com/openshift/openshift-docs/pull/46866 (for Version 4.7)

https://github.com/openshift/openshift-docs/pull/46867 (for Version 4.8)

The original text has already been LGTM'd by SME and QE, and Peer Review comments have been addressed.

### CODE REVIEW

**Review by kevchu3 — COMMENTED** [review]
LGTM!

**Comment by pamoedom** [comment]
LGTM

File: modules/ipi-preparing-reinstall-cluster-bare-metal.adoc [hunk 1]
```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+// //installing/installing_bare_metal_ipi/installing_bare_metal_ipi/ipi-install-installation-workflow.adoc
+
+[id="ipi-preparing-reinstall-cluster-bare-metal_{context}"]
+
+= Preparing to reinstall a cluster on bare-metal
+Before you reinstall a cluster on bare-metal, you must perform cleanup operations.
+
+.Procedure
+. Remove or reformat the disks for the bootstrap, master, and worker nodes. If you are working in a hypervisor environment, you will need to re-add any disks you removed. You are now starting with clean disks for the bootstrap, master, and worker nodes.
```
Comment (michaelryanpeter) [inline modules/ipi-preparing-reinstall-cluster-bare-metal.adoc:10]:
```suggestion
. Remove or reformat the disks for the bootstrap, <control-plane>, and worker nodes. If you are working in a hypervisor environment, you must add any disks you removed. You are now starting with clean disks for the bootstrap, <control-plane>, and worker nodes.
```

- I think that the last sentence can be cut. It doesn't add anything, and it is one more thing to wrangle with the conscious language issue mentioned in the comment above. 
- The term _re-add_ is not in the Merriam-Webster dictionary. I could not find guidance on the term in the SSG or ISG.
- [ISG](https://www.ibm.com/docs/en/ibm-style?topic=grammar-verbs#tense) advises against using future tense.

File: modules/ipi-preparing-reinstall-cluster-bare-metal.adoc [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+. Delete the artifacts that the previous installation generated:
+[source,terminal]
+----
+$ cd ; /bin/rm -rf auth/ bootstrap.ign master.ign worker.ign metadata.json .openshift_install.log .openshift_install_state.json
+----
```
Comment (michaelryanpeter) [inline modules/ipi-preparing-reinstall-cluster-bare-metal.adoc:15]:
```suggestion
. Delete the artifacts that the previous installation generated:
+
[source,terminal]
----
$ cd ; /bin/rm -rf auth/ bootstrap.ign master.ign worker.ign metadata.json \
.openshift_install.log .openshift_install_state.json
----
```
Refer to the sections on [lists](https://github.com/openshift/openshift-docs/blob/f6e9d0618b5498bd459e47ed8932a76b747a83cb/contributing_to_docs/doc_guidelines.adoc#lists) and [breaking up long lines](https://github.com/openshift/openshift-docs/blob/f6e9d0618b5498bd459e47ed8932a76b747a83cb/contributing_to_docs/doc_guidelines.adoc#lists) in the OSDOCS guidelines.

File: modules/ipi-preparing-reinstall-cluster-bare-metal.adoc [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+[start=3]
```
Comment (michaelryanpeter) [inline modules/ipi-preparing-reinstall-cluster-bare-metal.adoc:16]: By using the  `+` you should be able to delete `[start=3]` here.

File: modules/ipi-preparing-reinstall-cluster-bare-metal.adoc [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+. Generate new manifests and Ignition config files. Refer to the topic “Creating the Kubernetes manifest and Ignition config files" in Additional Resources, below, for further information.
```
Comment (michaelryanpeter) [inline modules/ipi-preparing-reinstall-cluster-bare-metal.adoc:17]:
```suggestion
. Generate new manifests and Ignition config files. See “Creating the Kubernetes manifest and Ignition config files" for more information.
```

**Review by michaelryanpeter — COMMENTED** [review]
A few things: 
- `enterprise-4.5` is longer being updated.
- You will need to split this up into several PRs. One for `main` and `enterprise-4.9`+, and then for the `4.6` to `4.8` to address the conscious language issues described in the comments.
- Please squash to one commit.

Reach out if you have questions. Slack is the best way to get my attention in a timely manner.

File: modules/ipi-preparing-reinstall-cluster-bare-metal.adoc [hunk 5]
```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+// //installing/installing_bare_metal_ipi/installing_bare_metal_ipi/ipi-install-installation-workflow.adoc
+
+[id="ipi-preparing-reinstall-cluster-bare-metal_{context}"]
+
+= Preparing to reinstall a cluster on bare-metal
```
[the remark this answers is not in the document]
Comment (tmulquee) [reply modules/ipi-preparing-reinstall-cluster-bare-metal.adoc:6]: Done.

File: modules/ipi-preparing-reinstall-cluster-bare-metal.adoc (hunk 1 above)
Comment (tmulquee) [reply modules/ipi-preparing-reinstall-cluster-bare-metal.adoc:10]: Done.

File: modules/ipi-preparing-reinstall-cluster-bare-metal.adoc (hunk 2 above)
Comment (tmulquee) [reply modules/ipi-preparing-reinstall-cluster-bare-metal.adoc:15]: Done. Will revise standards.

File: modules/ipi-preparing-reinstall-cluster-bare-metal.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (tmulquee) [reply modules/ipi-preparing-reinstall-cluster-bare-metal.adoc:10]: Resolved for 4.9 onwards.
Will create separate PR for 4.8 and earlier.

File: installing/installing_bare_metal_ipi/ipi-install-installation-workflow.adoc [hunk 6]
```diff
@@ -83,7 +83,9 @@ include::modules/ipi-install-following-the-installation.adoc[leveloffset=+1]
 
 include::modules/ipi-install-verifying-static-ip-address-configuration.adoc[leveloffset=+1]
 
+include::modules/ipi-preparing-reinstall-cluster-bare-metal.adoc[leveloffset=+1]
+
 [role="_additional-resources"]
```
[the remark this answers is not in the document]
Comment (tmulquee) [reply installing/installing_bare_metal_ipi/ipi-install-installation-workflow.adoc:88]: Unique IDs created for AR headings.

File: modules/ipi-preparing-reinstall-cluster-bare-metal.adoc [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+
+[id="ipi-preparing-reinstall-cluster-bare-metal_{context}"]
+
+= Preparing to reinstall a cluster on bare metal
+Before you reinstall a cluster on bare metal, you must perform cleanup operations.
+
+.Procedure
+. Remove or reformat the disks for the bootstrap, <control-plane>, and worker nodes. If you are working in a hypervisor environment, you must add any disks you removed.
+. Delete the artifacts that the previous installation generated:
+
```
Comment (kalexand-rh) [inline modules/ipi-preparing-reinstall-cluster-bare-metal.adoc:12]:
```suggestion
+
```

Attaching the command to the step is more future-proof than using `[start=3]`.

Comment (tmulquee) [reply modules/ipi-preparing-reinstall-cluster-bare-metal.adoc]: Added + to topic.

File: modules/ipi-preparing-reinstall-cluster-bare-metal.adoc (hunk 4 above)
Comment (tmulquee) [reply modules/ipi-preparing-reinstall-cluster-bare-metal.adoc:18]: Done.

File: modules/ipi-preparing-reinstall-cluster-bare-metal.adoc (hunk 3 above)
Comment (tmulquee) [reply modules/ipi-preparing-reinstall-cluster-bare-metal.adoc]: Done.

File: modules/ipi-preparing-reinstall-cluster-bare-metal.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (tmulquee) [reply modules/ipi-preparing-reinstall-cluster-bare-metal.adoc]: Sorry - realized I need separate PRs for 4.6, 4.7, and 4.8.

**Review by kevchu3 — COMMENTED** [review]
LGTM!

**Review by kelbrown20 — COMMENTED** [review]
LGMT from peer review!

**Comment by jboxman-rh** [comment]
@tmulquee, it looks like this needs a manual cherry pick for 4.9 and 4.10.

**Pull request merged**